packages/oidc-client/src/lib/logout.request.ts

```diff
@@ -48,15 +49,42 @@ export function logoutµ({
   ).pipe(
     // Delete local token and return combined results
     Micro.flatMap(([sessionResponse, revokeResponse]) =>
-      Micro.promise(async () => {
-        const deleteRes = await storageClient.remove();
-        const deleteResponse = typeof deleteRes === 'undefined' ? null : deleteRes;
-        return {
-          sessionResponse,
-          revokeResponse,
-          deleteResponse,
-        };
-      }),
+      Micro.promise(() => storageClient.remove()).pipe(
+        Micro.flatMap((deleteRes) => {
+          const deleteResponse = typeof deleteRes === 'undefined' ? null : deleteRes;
+
+          let errorKey;
+          if (sessionResponse && 'error' in sessionResponse) {
+            errorKey = 'sessionResponse';
+          } else if (revokeResponse && 'error' in revokeResponse) {
+            errorKey = 'revokeResponse';
+          } else if (
+            deleteResponse &&
+            typeof deleteResponse === 'object' &&
+            'error' in deleteResponse
+          ) {
+            errorKey = 'deleteResponse';
+          }
```

Comment:
I know I'm the one who suggested this, but I'm now thinking this `errorKey` could be misleading. If more than one response has an error, we will only share the first in this key. We could convert this string into an array and iterate over the errors, but at that point, I think we lose the value.

So, I'm thinking we just remove the `errorKey`, but leave `error`. Thoughts?